tests/cast.py

```diff
@@ -0,0 +1,10 @@
+def f():
+    s: str
+    s = "lpython"
+    x: list[str]
+    x = list(s)
+    y: list[str]
+    y = ['a', 'b', 'c']
+    x = list(y)
```

Comment:
The verification is not happening whether the generated list is correct or now? Can you add some asserts for constant values here? Something like,

```
assert list("lpython") == ['l', 'p', 'y', 't', 'h', 'o', 'n']
```

Reply:
Covering empty strings, large strings, single character strings would be good.